packages/common-db/index.js

```diff
@@ -1 +1,5 @@
-module.exports = require('./lib/common/tools');
\ No newline at end of file
+module.exports = {
+    tools: require('./lib/common/tools'),
+    getStatesObject: require('./lib/common/states'),
+    getObjectsObject: require('./lib/common/objects')
```

Comment:
```suggestion
    getStatesConstructor: require('./lib/common/states'),
    getObjectsConstructor: require('./lib/common/objects')
```

Reply:
We do export the constructor, right?

Reply:
Yes, otherwise code will be executed on requiring which will fail in initial setups where config is not created via setup first yet

Reply:
Constructor is fine for more

Reply:
In fact we export the "Object" :-) but all good